universal-application-tool-0.0.1/test/support/TestQuestionBank.java

```diff
@@ -266,6 +248,17 @@ private Question applicantFavoriteColor(QuestionEnum ignore) {
   private Question maybeSave(QuestionDefinition questionDefinition) {
     Question question = new Question(questionDefinition);
     if (canSave) {
+      // This odd way of finding the active version is because this class is
+      // entirely static and doesn't have room for injection of the Version repository,
+      // because it needs to work without the database.
```

Comment:
Thought (non-blocking): I think someone is either in-process or already has changed this class to use instance state

Reply:
fun fact, I drafted a ToTT about avoiding static context in Java and it was mostly well-received by the review committee but one person vetoed it 😞

Reply:
Yeah - I could refactor it again to inject the version repository, but I'm just straight up not gonna do it, ha.  I will update the comment so it's true now, though!